rules/sbom.bzl

```diff
@@ -0,0 +1,159 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""SBOM generation"""
```

Comment:
@aiuto question about usability. If I wanted to update this tool to include a transitive dependencies Label in the sBOM, how would I do that? For example, a program dependeds on Guava, and I want the sBOM to include the label of where Guava was pulled from `@maven//:com_google_guava_guava`,

Reply:
We need to get the coursair library from rules_jvm_external to pull that up. I don't have a design at hand for that today.